src/App.js

```diff
@@ -1,8 +1,46 @@
 import React from 'react';
 import './App.css';
+import CharacterInfo from './Components/Character/CharacterInfo';
+import ActionsButton from './Components/Actions/ActionsButton';
+import EnergyButton from './Components/Energy/EnergyButton';
+import CoinsButton from './Components/Coins/CoinsButton';
+import SeenButton from './Components/Seen/SeenButton';
+import CraftingButton from './Components/Crafting/CraftingButton';
+import Map from './Components/Map/Map';
+import DisastersContainer from './Components/Disasters/DisastersContainer';
 
-function App() {
-  return <div className="App">Project created</div>;
+class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
```

Comment:
We usually keep `App` component clean and render some kind of `MainScreen` component there.
This will be more convenient when you will use redux and wrap your main component with `store`.
Of course, you don't have to do it now. Just a suggestion.

Reply:
Good to know, i guess it will happen in near future